src/app/layout.tsx

```diff
@@ -72,13 +74,28 @@ const pendoInclude = `
               },
             });
         })("${process.env.REACT_APP_PENDO_API_KEY}");
-      }`;
+      }
+`;
 
 export default function RootLayout({
   children,
 }: Readonly<{
   children: React.ReactNode;
 }>) {
+  const [header, setHeader] = useState<string | undefined>();
```

Comment:
We need to fix this using actual web components. This is bad. Unless the header and footer are not web components but ESI. In which case we are done for.

Reply:
AFAIK they are ESI tags. There's webcomponent for these partials, but they are missing the content.

Reply:
😿